packages/@aws-cdk/aws-servicecatalog/test/integ.portfolio.expected.json

```diff
@@ -1,87 +1,184 @@
 {
-    "Resources": {
```

Comment:
What happened with this file? Why are there so many changes here?

Reply:
need to look into this, seems like some git churn even on resources that weren't modified and are still there.

Reply:
I think originally did not include the updated integ test file that created these changes.